rgw/v2/tests/s3cmd/test_bucket_stats.py

```diff
@@ -0,0 +1,132 @@
+"""
+test_s3cmd - Test bucket stats consistency in number of objects using s3cmd
+
+Usage: test_bucket_stats.py -c <input_yaml>
+
+<input_yaml>
+    Note: Following yaml can be used
+    test_bucket_stats.yaml
+
+Operation:
+    Create an user
+    Create a bucket with user credentials
+    Performs multipart upload with object name consist of whitespace
+    checks for bucket stats consistency among number of objects
+"""
+
+
+import argparse
+import json
+import logging
+import os
+import sys
+import traceback
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+
+
+from v2.lib import resource_op
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, IOInfoInitialize
+from v2.lib.s3cmd import auth as s3_auth
+from v2.tests.s3cmd import reusable as s3cmd_reusable
+from v2.utils import utils
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def test_exec(config):
+    """
+    Executes test based on configuration passed
+    Args:
+        config(object): Test configuration
+    """
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    user_info = resource_op.create_users(no_of_users_to_create=config.user_count)[0]
+    user_name = user_info["user_id"]
+
+    ip_and_port = s3cmd_reusable.get_rgw_ip_and_port()
+    s3_auth.do_auth(user_info, ip_and_port)
+
+    if config.bucket_stats is True:
+        bucket_name = utils.gen_bucket_name_from_userid(user_name, rand_no=0)
+        s3cmd_reusable.create_bucket(bucket_name)
+        log.info(f"Bucket {bucket_name} created")
+        utils.exec_shell_cmd(f"fallocate -l 25m obj25m")
+        object_name = f"s3://{bucket_name}/encyclopedia/space & universe/.bkp/journal$i"
+        cmd = (
+            "for i in {1..200}; do /home/cephuser/venv/bin/s3cmd put obj25m "
```

Comment:
Can we add object_count = 200 parameter in yaml file so that we can change object count value in future if required

Reply:
done